web/src/components/Storyblok/News/NewsCard/NewsCard.tsx

```diff
@@ -42,14 +48,18 @@ export const NewsCard: React.FC<NewsCardProps> = ({
 			>
 				{" "}
```

Comment:
I think this is related to stuff I added originally - I used aria-hidden on the Link component for this reason, as otherwise we'd just be repeating the link below & can assume that the image is purely decorative

Reply:
Yep I haven't done much work here apart from to add the 'isNewsListItem' variant and styling